sr_utilities/scripts/sr_utilities/calibrate_hand_finder.py

```diff
@@ -45,9 +45,11 @@ def generate_controllers():
 def main():
     if rospy.is_shutdown():
         return
+
+    rospy.init_node('calibration', anonymous=True)
+
```

Comment:
This is just reverting a recent change in order.This is the right order (first initialize the node, then do the rest).